src/scripts/sync_subclassof.py

```diff
@@ -304,11 +308,13 @@ def sync_subclassof(
     #  after. It could be that I invalidated the cache somehow without realizing it. Rerunning cache fixed it though.
     missing_ancestor_rels = rels_indirect_mondo_mondo.union(rels_direct_mondo_mondo).difference(ancestors_mondo_mondo)
     if missing_ancestor_rels:
-        raise RuntimeError(
```

Comment:
Can you @joeflack4 review the current case? There are seemingly 11 relationships including

```
 Intersection (Top 5): [('MONDO:0014937', 'rdfs:subClassOf', 'MONDO:0003008'), ('MONDO:0014938', 'rdfs:subClassOf', 'MONDO:0002466'), ('MONDO:0011790', 'rdfs:subClassOf', 'MONDO:0100500'), ('MONDO:0014937', 'rdfs:subClassOf', 'MONDO:0002466'), ('MONDO:0100534', 'rdfs:subClassOf', 'MONDO:0006260')]
```

in that list. I would suggest:

1. Create a new issue about this, which suggests changing this back to `raise RuntimeError`.
2. Determining why the 11 cases occur and fix the route cause.

Fixes in new PR.

Reply:
Hmm, I was hoping this error would never happen again.

Sure, I'll do that. "Fixes in new PR" sounds like part of (2). Is "new PR" one that is currently open or not one that exists yet?

Are you asking me just to review the case, which seems like tasks (1). I assume you are also asking me to take a stab at (2) as well?

Reply:
Will answer in PR later